### DISCUSSION TITLE
[v2.0.56] help file / explanation new template variables?

### DISCUSSION THREAD

**Discussion by emp-00** [gildas-lormeau/single-file-cli#115] [Q&A]
Dear @gildas-lormeau - referring to the latest release:
_added "{options-json}", "{options-text}" variables, and "%option-value" function in templates_

Where can I find documentation / help how to use the new options? I checked the [help html file here](https://github.com/gildas-lormeau/SingleFile/blob/1e60e1d2e6e89ad636fbe40de305413f1d0be1a8/src/ui/pages/help.html) but I cannot find these options? Thanks for a quick clarification.

**Comment by gildas-lormeau** [comment]
Thank you for pointing out this omission. In fact, I had written it, but apparently I made a mistake and lost everything. I've updated the help page, which will be available in the next version (v2.0.62). See https://github.com/gildas-lormeau/SingleFile/blob/bc2c6356fbdaa1ced615c8b6542ec21e32e5f39a/src/ui/pages/help.html#L991-L994 and https://github.com/gildas-lormeau/SingleFile/blob/bc2c6356fbdaa1ced615c8b6542ec21e32e5f39a/src/ui/pages/help.html#L1183-L1185.

**Reply by emp-00** [reply]
Thanks for taking this up (sounds like a big version jump from .56 to .62 is coming? Cool! :-) )

For me it is still not clear, which "options" this refers to. Do I get it right, these are actually options of the "single-file" **browser addon**? How is this helpful for the single-file-cli command line tool? You are describing them as "options to save the page" - maybe it would be helpful to add a more concrete example or at least a clear statement _which options_ are available? Currently it's very general e.g. "option1: value1" which leaves me unsure what kind of options you mean. Sorry for asking such stupid questions but I still don't know how this new feature works in practice. No hurries, just would like to understand if I can use this or not.

```
{options-json}: the JSON representation of the options used to save the page
{options-text}: the text representation of the options used to save the page. 
The text representation is formatted with a list of options and their values separated by a line break 
(e.g. "option1: value1\noption2: value2")
```

**Reply by gildas-lormeau** [reply]
Thank you for the feedback, I updated the help page and added some concrete examples.
